scss/_body_aside.scss

```diff
@@ -0,0 +1,81 @@
+/// Style for big-numbers
+@mixin oTypographyBigNumber {
```

Comment:
For our transformation of big numbers for article, we're planning to wrap them in an aside, hence why I had them here. A big number should always be complementary to the article body.